fiona/collection.py

```diff
@@ -140,14 +141,8 @@ def __init__(self, path, mode='r', driver=None, schema=None, crs=None,
             self._schema = schema
 
             self._check_schema_driver_support()
-
-            if crs_wkt:
-                self._crs_wkt = crs_wkt
-            elif crs:
-                if 'init' in crs or 'proj' in crs or 'epsg' in crs.lower():
-                    self._crs = crs
-                else:
-                    raise CRSError("crs lacks init or proj parameter")
+            if crs_wkt or crs:
+                self._crs_wkt = crs_to_wkt(crs_wkt or crs)
```

Comment:
👍

Reply:
There are still a few places where `self._crs` is used no? (and with the above, this is no longer set?)

Reply:
That's okay. It is a property that defaults to None:
```python
    @property
    def crs(self):
        """Returns a Proj4 string."""
        if self._crs is None and self.session:
            self._crs = self.session.get_crs()
        return self._crs
```

Reply:
Ah. found what you are talking about.

Reply:
Removed redundant code.